WinUIGallery/ControlPages/SystemBackdropsPage.xaml

```diff
@@ -19,45 +19,71 @@
       mc:Ignorable="d">
 
     <StackPanel>
-        <local:ControlExample HeaderText="Create a window with a built-in Mica system backdrop."
-                              CSharpSource="SystemBackdrops\SystemBackdropsSample3.txt">
-            <local:ControlExample.Xaml>
-                <x:String xml:space="preserve">
-// Option 1 - implement Mica with Xaml.
-&lt;Window.SystemBackdrop&gt;
-    &lt;MicaBackdrop /&gt;
-&lt;/Window.SystemBackdrop&gt;
-                </x:String>
-            </local:ControlExample.Xaml>
-            <Button Click="createBuiltInMicaWindow_Click">Create Window with built-in Mica</Button>
+        <local:ControlExample HeaderText="MicaBackdrop:"
+                              CSharpSource="SystemBackdrops\SystemBackdropsSample1_cs.txt"
+                              XamlSource="SystemBackdrops\SystemBackdropsSample1_xaml.txt">
+            <StackPanel>
+                <TextBlock TextWrapping="WrapWholeWords">
+                    Inheritance: Object > DependencyObject > SystemBackdrop <LineBreak/> <LineBreak/>
+                    
+                      Mica is an opaque, dynamic material that incorporates theme and desktop wallpaper to paint the background of windows.
```

Comment:
I think having one sample that includes the default ways of including both Mica and Acrylic is better here. For the custom versions, having it split up made sense since its a lot more involved.